kpops/component_handlers/helm_wrapper/model.py

```diff
@@ -87,6 +87,11 @@ class YamlReader:
     def __iter__(self) -> Iterator[str]:
         # discard all output before template documents
         index = self.content.index("---")
-        self.content = self.content[index:-1]
+        notes = "\n\nNOTES:\n"
```

Comment:
how about making this a constant variable (uppercase) and defining it outside the method. Since it is not changing it doesn't have to live inside the method

Reply:
Makes sense, done